package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@harvard-lts/mirador-pdiiif-plugin",
-  "version": "0.1.4",
+  "version": "0.1.5",
```

Comment:
I see that mps-viewer is using 0.1.6 and I see 0.1.6 in npmjs.com but it doesn't look that the version has been updated in this PR. I suspect you just didn't commit it after building it to npmjs.com....

Reply:
Hmm that's weird, I suspect you're right. Will look at this, but I might make another change for one of your other comments that bumps this up again